js/webui.js

```diff
@@ -617,7 +633,8 @@ var theWebUI =
 				}
          		}
 		});
-		newSettings["webui.lang"] = GetActiveLanguage();
+		if ($type(newSettings["webui.lang"]))
```

Comment:
This is a strange condition. Why we need it? 
If key 'webui.lang' is absent, then we must create it.

Reply:
We need it for safety reasons. Otherwise, the WebUI will not initialize if `getsettings.php` sends an empty JSON string.

Reply:
Sorry, I can't understand. 
For example, ```getsettings.php``` sends an empty JSON string, i.e. ```{}```.
In the ```newSettings``` we will have an empty object. 
Why we can't create new key for this object with a language setting?

Reply:
Okay this problem has been resolved.